### PR TITLE
Add pop up mobile menu

### PR DESCRIPTION
**Pull request by Lucas-Masaba**
Add javascript linters
Add javascript file
Add mobile menu section to HTML
Add CSS transition
Add javascript events to mobile menu

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -623,12 +623,99 @@ form {
...
+
+  .mobile-menu {
+    position: fixed;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    right: 0;
+    background: #6070ff;
+    mix-blend-mode: multiply;
+    backdrop-filter: blur(8px);
```
Comment (bini-i) [inline style.css:653]: - [ ] backdrop-filter is not available on firefox and it is causing the mobile menu background to differ from the design on firefox as shown below:
- Your page
image

- Figma design
image

> Hint: When the mobile menu is active, you can set the filter property for the background elements as `filter: blur(4px);` to achieve the same result. Kindly refer [here](https://stackoverflow.com/questions/38145368/css-workaround-to-backdrop-filter) for more.

**Review by bini-i — CHANGES_REQUESTED** [review]
# Status: Changes Required :yellow_circle:            
### Hello @Lucas-Masaba  :wave: 

Your project looks good and shows you have put a lot of effort into it. Kindly work on the following suggestions to further improve your project.
## Summary
- Awesome README and pull request description
- Linters and github workflow are correctly setup
-  Hamburger menu is functional
## Optional suggestions
-   [_OPTIONAL_]

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

#### If you have any questions feel free to ask within the PR thread.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
### HAPPY CODING :computer: :blush: 

> #### Reviewed by Binyam Hailemeskel (Code Reviewer)
> GitHub: [@bini-i](https://github.com/bini-i) 
> Twitter: [@binyamshewa](https://twitter.com/binyamshewa)
> LinkedIn: [LinkedIn](https://www.linkedin.com/in/binyam-hailemeskel-728048151/)

File: index.html [hunk 2]
```diff
@@ -5,20 +5,38 @@
...
+        About
+      </a>
+      <a href="#contact_form" data-close-button class="navigate">
+        Contact
+      </a>
+    </nav>
+  </div>
   <header>
     <div class="header-nav">
       <a href="#" class="my-logo">My Logo</a>
```
Comment (bini-i) [inline index.html:32]: - [ ] Please make sure to maintain the same margin and position of elements as shown in the design. Currently, the logo is flushed against the left edge of the screen unlike the design, which has margins around it as shown below:

- Your page
image

- Figma design
image

File: style.css (hunk 1 above)
Comment (Lucas-Masaba) [reply style.css:653]: Please give me more insights on this. I have failed to find a solution.

**Review by rotimiazeez — APPROVED** [review]
# Hi @Lucas-Masaba   🎉, 

### Your project is complete👏👏! You have implemented the changes🕺... it's time to merge it :shipit: 🚀
Congratulations! 🎉 👍

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**